paprika/utils.py

```diff
@@ -11,6 +11,19 @@
 logger = logging.getLogger(__name__)
 
 
+def get_key(dct, value):
+    """Get dictionary key given the value"""
+    return [key for key in dct if (dct[key] == value)]
```

Comment:
If there are multiple keys that have a given value, does this just return the first one? (I'm not sure this is a problem.)

```python
d = {
  "a": 2
  "b": 2
  }
```

I think this would just return `a`, right?

Reply:
Looks like this will return both `a` and `b` but I'm only going to use this function for a dictionary with unique values. I'll make a note on the docstring about this and maybe give a warning and/or raise an exception if there are more than one key with the same values.

Reply:
Oh right, that makes sense. Yup, good thinking about the note/warning.